app/controllers/scans_controller.rb

```diff
@@ -27,20 +27,11 @@ def new_request_params
   end
 
   def redirect_to_success_with_token
-    redirect_to illiad_query(current_request)
+    redirect_to illiad_url
```

Comment:
The user will not have to do anything special if all goes correctly. The rft.genre=scananddeliver parameter instructs illiad to use a specific set of html pages. On those pages exist javascript to automatically submit the form (hence modeling the request in illiad) and upon success take them back to the callback (successful_scan) url.